regen-defs.zsh

```diff
@@ -23,8 +23,14 @@ for folder in $folders; do
     stdout_file=$(mktemp)
     stderr_file=$(mktemp)
 
+    # Cleanup on interruption
+    cleanup() {
+      rm "$stdout_file" "$stderr_file"
+    }
+    trap cleanup INT TERM
+
     # Execute the command and capture stdout and stderr
-    uv run "$cpplint" $cmd > "$stdout_file" 2> "$stderr_file"
+    eval uv run "$cpplint" $cmd > "$stdout_file" 2> "$stderr_file"
```

Comment:
How does this change things?

Reply:
It makes globbing work, since otherwise we literally pass "*" to cpplint, at least on my machine.